test/spec/features/service/link_spec.rb

```diff
@@ -1,44 +1,40 @@
 describe 'service link' do
   after(:each) do
-    %w(test-1 test-2).each do |s|
-      run "kontena service rm --force #{s}"
-    end
+    run "kontena service rm --force test-1"
+    run "kontena service rm --force test-2"
     run "kontena stack rm --force simple"
   end
 
   it 'links service to target' do
     run "kontena service create test-1 redis:3.0"
     run "kontena service create test-2 redis:3.0"
-    k = run "kontena service link test-1 test-2"
-    expect(k.code).to eq(0)
-    k = run "kontena service show test-1"
+    run! "kontena service link test-1 test-2"
+    k = run! "kontena service show test-1"
     expect(k.out.match(/^\s+- test-2\s*$/)).to be_truthy
   end
 
   it 'links service to stack service' do
     with_fixture_dir("stack/simple") do
-      run 'kontena stack install --no-deploy'
+      run! 'kontena stack install --no-deploy'
     end
-    run "kontena service create test-1 redis:3.0"
-    k = run "kontena service link test-1 simple/redis"
-    expect(k.code).to eq(0)
-    k = run "kontena service show test-1"
+    run! "kontena service create test-1 redis:3.0"
+    run! "kontena service link test-1 simple/redis"
+    k = run! "kontena service show test-1"
     expect(k.out.match(/^\s+\- simple\/redis\s*$/)).to be_truthy
   end
 
   it 'links stack service with existing links to service' do
     with_fixture_dir("stack/links") do
-      run 'kontena stack install --no-deploy links.yml'
+      run! 'kontena stack install --no-deploy links.yml'
     end
-    run "kontena service create test-1 redis:3.0"
-    k = run "kontena service link simple/bar test-1"
-    expect(k.code).to eq(0)
-    k = run "kontena service show simple/bar"
+    run! "kontena service create test-1 redis:3.0"
+    run! "kontena service link simple/bar test-1"
+    k = run! "kontena service show simple/bar"
     expect(k.out.match(/^\s+\- test-1\s*$/)).to be_truthy
```

Comment:
This spec needs to also unlink the service from `bar` so that it can get removed...

```
 [fail] Removing service test-1      
 [error] 422 : Unprocessable Entity:
	service: Cannot delete service that is linked to another service (bar)
```

Reply:
👍